src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+import java.time.LocalDate;
+
+public class Task {
+    protected String msg;
+    protected String status;
```

Comment:
Interesting use of String instead of Boolean! I just realized how duke doesn't make use of this variable other than in printing marks, so it makes sense.